xml/telnet_banners.xml

```diff
@@ -0,0 +1,648 @@
+      NjbyBDUCkgaXMgaW5zdGFsbGVkIG9uIHRoaXMgZGV2aWNlLiANClRoaXMgZmVhdHVyZSByZXF
+      1aXJlcyB0aGUgb25lLXRpbWUgdXNlIG9mIHRoZSB1c2VybmFtZQo=
+    </example>
+    <param pos="0" name="service.vendor" value="Cisco"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+    <param pos="0" name="os.family" value="IOS"/>
+    <param pos="0" name="os.product" value="IOS"/>
+    <param pos="0" name="os.device" value="Router"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:cisco:ios:-"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="0" name="hw.device" value="Router"/>
+  </fingerprint>
+  <fingerprint pattern="^192.0.0.64 login:\s*$">
+    <description>Hikvision cameras and NVRs (multiple)</description>
+    <example>192.0.0.64 login:</example>
+    <param pos="0" name="service.vendor" value="Hikvision"/>
+    <param pos="0" name="os.vendor" value="Hikvision"/>
+    <param pos="0" name="hw.vendor" value="Hikvision"/>
+  </fingerprint>
+  <fingerprint pattern="^Remote Management Console\r\nlogin:\s*$">
+    <description>Juniper Netscreen</description>
+    <!-- Remote Management Console\r\nlogin: -->
+    <example _encoding="base64">UmVtb3RlIE1hbmFnZW1lbnQgQ29uc29sZQ0KbG9naW46Cg==</example>
+    <param pos="0" name="service.vendor" value="Juniper"/>
+    <param pos="0" name="service.family" value="NetScreen"/>
+    <param pos="0" name="service.product" value="NetScreen"/>
+    <param pos="0" name="os.vendor" value="Juniper"/>
+    <param pos="0" name="os.device" value="Firewall"/>
+    <param pos="0" name="os.family" value="ScreenOS"/>
+    <param pos="0" name="os.product" value="ScreenOS"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:juniper:screenos:-"/>
+    <param pos="0" name="hw.vendor" value="Juniper"/>
+    <param pos="0" name="hw.device" value="Firewall"/>
+    <param pos="0" name="hw.product" value="NetScreen"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\r|\n)*KWS-1043N login:\s*$">
+    <description>Clipcomm KWS router</description>
+    <example>KWS-1043N login:</example>
+    <param pos="0" name="service.vendor" value="Clipcomm"/>
+    <param pos="0" name="service.product" value="telnet"/>
+    <param pos="0" name="hw.vendor" value="Clipcomm"/>
+    <param pos="0" name="hw.device" value="Router"/>
+    <param pos="0" name="hw.product" value="KWS-1043N"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\r|\n)*(SMCD3\w+-\w\w\w) login:\s*$">
+    <description>SMC Cable Modem</description>
+    <example>SMCD3GN2-BIZ login:</example>
+    <param pos="0" name="service.vendor" value="SMC Networks"/>
+    <param pos="0" name="service.product" value="telnet"/>
+    <param pos="0" name="hw.vendor" value="SMC Networks"/>
+    <param pos="0" name="hw.device" value="Cable Modem"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+  <fingerprint pattern="^Welcome to Microsoft Telnet Service \r\n\n\rlogin:\s*$">
+    <description>Microsoft Windows</description>
```

Comment:
CE

Reply:
Doh, actually, I think the CE reference is incorrect. Fixed